lib/wanda_web/schemas/execution/agent_check_error.ex

```diff
@@ -3,23 +3,30 @@ defmodule WandaWeb.Schemas.ExecutionResponse.AgentCheckError do
 
   alias OpenApiSpex.Schema
 
-  alias WandaWeb.Schemas.ExecutionResponse.{ExpectationEvaluation, Fact}
+  alias WandaWeb.Schemas.ExecutionResponse.{Fact, FactError}
 
   require OpenApiSpex
 
   OpenApiSpex.schema(%{
     title: "AgentCheckError",
-    description: "The result of check on a specific agent",
+    description:
+      "An error describing that some of the facts could not be gathered on a specific agent",
     type: :object,
     properties: %{
       agent_id: %Schema{type: :string, format: :uuid, description: "Agent ID"},
-      facts: %Schema{type: :array, items: Fact, description: "Facts gathered from the targets"},
-      expectation_evaluations: %Schema{
+      facts: %Schema{
+        description: "Facts gathered, possibly with errors, from the target",
```

Comment:
This facts list is nullable in our code

Reply:
Interesting. Can you point out where this happens? I am not finding it :sweat_smile: